src/store.js

```diff
@@ -20,6 +20,7 @@ export const store = reactive({
 	// Metadata
 	metaDataItem: false,
 	metaDataList: [],
+	metadataDataKey: false,

```

Comment:
Why not metaDataPropertyItem ipv key? dan kan je hem in een keer injecten en heb je een fetch minder bij het openen van het object.